### PR TITLE
InfiniteLoopDocumentTokenSourceTest Test class added

### PR DESCRIPTION
**Pull request by sam974**
- one test case fallen in a frozen state
- and the other one is successful

### CODE REVIEW

File: org.xtext.example.mydsl.ui.tests/src/org/xtext/example/mydsl/ui/tests/InfiniteLoopDocumentTokenSourceTest.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+			}
+			
+			@Override
+			public void documentAboutToBeChanged(DocumentEvent event) {
+			}
+		});
+	}
+	
+	
+	@Test
```
Comment (miklossy) [inline org.xtext.example.mydsl.ui.tests/src/org/xtext/example/mydsl/ui/tests/InfiniteLoopDocumentTokenSourceTest.java]: Could you add 2 seconds `@Test(timeout = 2000) ` as a maximum time window for the test execution?

Comment (sam974) [reply org.xtext.example.mydsl.ui.tests/src/org/xtext/example/mydsl/ui/tests/InfiniteLoopDocumentTokenSourceTest.java]: Very good idea, thank you!
